spring-integration-xmpp/src/main/java/org/springframework/integration/xmpp/config/XmppConnectionFactoryBean.java

```diff
@@ -164,11 +163,13 @@ public void start() {
 			try {
 				this.connection.connect();
 				this.connection.addConnectionListener(new LoggingConnectionListener());
-				this.connection.login();
+				Roster roster = Roster.getInstanceFor(this.connection);
 				if (this.subscriptionMode != null) {
-					Roster.getInstanceFor(this.connection)
-							.setSubscriptionMode(this.subscriptionMode);
+					roster.setSubscriptionMode(this.subscriptionMode);
+				} else {
```

Comment:
Our code style is like `else` from a new line after `}`.

You can catch it pretty easy with the `./gradlew check` command.